CHANGELOG.md

```diff
@@ -1,9 +1,175 @@
 # Chef Server Changelog
 
-# 12.2.1 (pending)
+# 12.3.0 (2015-11-12)
 
 ## Updated components
   * postgresql (9.2.10 -> 9.2.14)
+  * rubygems (1.8.24 -> 2.4.5)
+  * bundler (1.5.3 -> 1.10.6)
+  * postgresql92 (9.2.10 -> 9.2.14)
```

Comment:
minor nitpick: Looks like Postgres is listed twice - once as `postgresql` and once as `postgresl92'